### DISCUSSION TITLE
Twilio Sans Mono

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#2408] [Office hours] [answered]
Hi team, 

Serverless team is planning to update TwiML Bins UI currently to move it to Paste and as part of it we are swapping our code editor(Monaco). It would be a great opportunity to see how it works with the new font, **Twilio Sans Mono**.

Do you have any details on how we can use it in the browser? 

We reached out to brand team and they told us check in with Paste team.

**Comment by TheSisb** [comment, marked as answer]
Hi @sang-yeo,

We won't be rolling out the new font until the One Twilio BPM wraps up and design refresh is completed. We'll make a big announcement when we do, so you'll be looped in!

In the meantime, I believe the font is installed on all of our machines, so you may be able to test it locally by writing a CSS font-family override in your browser's inspector tool, as it will load the font from your machine.
![image](https://user-images.githubusercontent.com/1592327/168160733-7c096ecf-2dc9-4a6e-b410-6e4927a8301e.png)


Lastly, for my own curiosity, what will you be replacing Monaco with?

**Reply by sang-yeo** [reply]
Thanks for the update @TheSisb! Adding my PM here for his visibility. @dkundel 

TwiML Bins currently using the codemirror editor. 

Do you know the timeline for the One Twilio BPM?

**Reply by dkundel** [reply]
Thanks @TheSisb! Since you all don't have a code editor component our thought was that we could possibly just swap the Monaco default font with Twilio Sans Mono without touching any other Paste components. Any concerns?

**Reply by TheSisb** [reply]
> TwiML Bins currently using the codemirror editor.

Oh I see, yeah Monaco is great! That's good news!

> Do you know the timeline for the One Twilio BPM?

It will wrap up in Q4 2022, and the work to bring the font into Paste will probably be in Q4 too.

> Since you all don't have a code editor component our thought was that we could possibly just swap the Monaco default font with Twilio Sans Mono without touching any other Paste components. Any concerns?

No concerns there. You may want to wait a little until all of Paste begins to use the new font for consistency, but it's your call.

**Reply by sang-yeo** [reply]
Thanks for your answer @TheSisb. We are super stoked to use the new font for TwiML Bins!
